class.jetpack-gutenberg.php

```diff
@@ -107,7 +107,7 @@ private static function prepend_block_prefix( $block_name ) {
 	 * @return string The unprefixed extension name.
 	 */
 	private static function remove_extension_prefix( $extension_name ) {
-		if ( wp_startswith( $extension_name, 'jetpack/' ) || wp_startswith( $extension_name, 'jetpack-' ) ) {
+		if ( 0 === strpos( $extension_name, 'jetpack/' ) || 0 === strpos( $extension_name, 'jetpack-' ) ) {
```

Comment:
Can you verify that this is expected to be case sensitive?

Reply:
This should be equivalent:

https://github.com/Automattic/jetpack/blob/50ad87a912be721859b10bf9b6bd195be138a914/functions.compat.php#L75-L79

I do expect case sensitive, these are expected to be lower case to the best of my knowledge.

Reply:
Awesome, thanks! :shipit: